### DISCUSSION TITLE
Занос автомобилей на зимней карте

### DISCUSSION THREAD

**Discussion by ghost** [octothorp-team/public-dobrograd#845] [Предложения]
Безусловно, это только на время зимы, но, однако, интересная функция. Предлагаю реализовать разные категории шин (Зимняя, летняя), чтобы их можно было менять. На летней шине зимой будет "очень весело" ездить, т.к автомобиль будет заносить и тормозить будет тяжело. Поменяв шину на зимнюю - заносов не будет. Больше реализма, и автомеханику хоть какой-то плюс денег.

**Comment by damirmd** [comment]
Я уже предлагал это, но идея хорошая! 
[https://github.com/octothorp-team/public-dobrograd/discussions/809](url)
